scripts/commands/updateApiDocs.ts

```diff
@@ -61,14 +66,35 @@ type Pkg = {
     collapsed?: boolean;
     nestModule?(id: string): boolean;
   };
-  transformLink?: (
-    url: string,
-    text?: string,
-  ) => { url: string; text?: string } | undefined;
+  transformLink?: (url: string, text?: string) => Link | undefined;
 };
 
 type PkgHtml = { pkg: Pkg; version: string; path: string };
 
+function transformLink(
+  url: string,
+  text: string | undefined,
```

Comment:
ditto on the name `text`. 

Also FYI in TS, you can use `text?: string` I believe for this same syntax. Either style is fine to me.

Reply:
that's neater; thanks!